public/app/plugins/panel/dashlist/DashList.tsx

```diff
@@ -107,13 +107,15 @@ export function DashList(props: PanelProps<Options>) {
   const dispatch = useDispatch();
   const navIndex = useSelector((state) => state.navIndex);
 
+  const throttledRenderCount = useThrottle(props.renderCounter, 5000);
+
   useEffect(() => {
     fetchDashboards(props.options, props.replaceVariables).then((dashes) => {
       setDashboards(dashes);
     });
-  }, [props.options, props.replaceVariables, props.renderCounter]);
+  }, [dashboards, props.options, props.replaceVariables, throttledRenderCount]);
```

Comment:
why's `dashboards` in here now? Seems this would still cause an infinite request 'loop'?

Reply:
I was doing a deep equal to prevent rerenders for same dashboards list. I'll remove it from the dependency list since I also removed the check 🤦‍♂️